lib/ansible/module_utils/network/solace/solace_utils.py

```diff
@@ -215,14 +243,15 @@ def _parse_bad_response(resp):
 
 def _make_request(func, solace_config, path, json=None):
     if func is requests.get:
-        path += '?count=' + str(MAX_REQUEST_ITEMS)
+        """ path += '?count=' + str(MAX_REQUEST_ITEMS) """
```

Comment:
was this deliberate? Do you have an alternative way of returning more than the default (10?) results?

Reply:
There are resources which do not support the "count" query parameter.
I made this configurable via a class variable instead. I'll send a new pull request.